yt_dlp/extractor/generic.py

```diff
@@ -876,17 +876,19 @@ class GenericIE(InfoExtractor):
         # Wistia embed
         {
             'url': 'http://study.com/academy/lesson/north-american-exploration-failed-colonies-of-spain-france-england.html#lesson',
-            'md5': '1953f3a698ab51cfc948ed3992a0b7ff',
+            'md5': 'b9676d24bf30945d97060638fbfe77f0',
             'info_dict': {
-                'id': '6e2wtrbdaf',
-                'ext': 'mov',
-                'title': 'paywall_north-american-exploration-failed-colonies-of-spain-france-england',
-                'description': 'a Paywall Videos video from Remilon',
-                'duration': 644.072,
+                'id': '5vd7p4bct5',
+                'ext': 'bin',
+                'title': 'md5:db27290a04ae306319b0b5cce3cdf7bd',
+                'description': 'md5:e835b7808e11aaef29ccdc28888437af',
+                'duration': 623.019,
                 'uploader': 'study.com',
-                'timestamp': 1459678540,
-                'upload_date': '20160403',
-                'filesize': 24687186,
+                'timestamp': 1663258727,
+                'upload_date': '20220915',
+                'filesize': 29798093,
+                'age_limit': 0,
+                'thumbnail': r're:^https?://.+\.jpg$',
```

Comment:
Any clue why this changed?

Reply:
as far as the format/filesize/etc, Wistia now uses a `.bin` extension for their best quality format. mp4 is still available, but no longer the best quality.

not sure why the ID and metadata changed, possibly a site revamp